### PR TITLE
deprecate at least in container-chain nodes, native executor

### PR DESCRIPTION
**Pull request by girazoki**
We have seen a lot of people customizing the template runtime but not changing the spec name. This makes the full-nodes use the native compiled runtime and fail importing blocks.

I am not a big fan of doing it, but since it's the trend that comes from polkadot-sdk (see https://github.com/paritytech/polkadot-sdk/pull/4329) I guess we should start using only the wasm-executor

### CODE REVIEW

**Comment by bkchr** [comment]
> I am not a big fan of doing

Why? :D

**Comment by girazoki** [comment]
> > I am not a big fan of doing
> 
> Why? :D

Just because we lose a bit of efficiency in our tests and those chains we have implementing the template untouched. But I guess it's more a bit of paranoia than anything else

File: container-chains/nodes/frontier/src/service.rs [hunk 1]
```diff
@@ -18,7 +18,6 @@
 
 #[allow(deprecated)]
 use {
-    crate::client::TemplateRuntimeExecutor,
```
Comment (ParthDesai) [inline container-chains/nodes/frontier/src/service.rs:21]: We need to remove this native executor as well right?

**Review by ParthDesai — CHANGES_REQUESTED** [review]

**Comment by bkchr** [comment]
Okay :)

File: container-chains/nodes/frontier/src/service.rs (hunk 1 above)
Comment (girazoki) [reply container-chains/nodes/frontier/src/service.rs:21]: yep

Comment (girazoki) [reply container-chains/nodes/frontier/src/service.rs:21]: Updated

**Review by ParthDesai — APPROVED** [review]

**Pull request merged**